### PR TITLE
[SEDONA-112] Add ST_AsEWKT

### PR DESCRIPTION
**Pull request by pavan02**
## Is this PR related to a proposed Issue?
[SEDONA-112](https://issues.apache.org/jira/projects/SEDONA/issues/SEDONA-112)

## What changes were proposed in this PR?
ST_AsEWKT Function

## How was this patch tested?
Yes I have added unit tests

## Did this PR include necessary documentation updates?
Yes.

### CODE REVIEW

**Comment by pavan02** [comment]
@jiayuasu Could you please re-run python build? It ran fine in my local. Thanks.

File: flink/src/test/java/org/apache/sedona/flink/FunctionTest.java [hunk 1]
```diff
@@ -73,10 +73,23 @@ public void testPointOnSurface() {
...
     public void testReverse() {
         Table polygonTable = createPolygonTable(1);
         Table ReversedTable = polygonTable.select(call(Functions.ST_Reverse.class.getSimpleName(), $(polygonColNames[0])));
         Geometry result = (Geometry) first(ReversedTable).getField(0);
         assertEquals("POLYGON ((-0.5 -0.5, 0.5 -0.5, 0.5 0.5, -0.5 0.5, -0.5 -0.5))", result.toString());
     }
+
+    /* ST_SetSrid Has not been implemented in flink yet
+    @Test
+    public void testAsEWKT() {
```
Comment (jiayuasu) [inline flink/src/test/java/org/apache/sedona/flink/FunctionTest.java:86]: Please enable the test case. Why do you call it ReverseTable?

File: docs/api/flink/Function.md [hunk 2]
```diff
@@ -150,4 +150,22 @@ FROM df
...
+
+Introduction: Return the Extended Well-Known Text representation of a geometry.
+EWKT is an extended version of WKT which includes the SRID of the geometry.
+The format originated in PostGIS but is supported by many GIS tools.
+If the geometry is lacking SRID a WKB format is produced.
+[Se ST_SetSRID](#ST_SetSRID)
+
+Format: `ST_AsEWKT (A:geometry)`
+
+Since: `v1.1.1`
```
Comment (jiayuasu) [inline docs/api/flink/Function.md:165]: v.1.2.1

File: docs/api/sql/Function.md [hunk 3]
```diff
@@ -302,6 +302,24 @@ SELECT ST_AsEWKB(polygondf.countyshape)
...
+
+Introduction: Return the Extended Well-Known Text representation of a geometry.
+EWKT is an extended version of WKT which includes the SRID of the geometry.
+The format originated in PostGIS but is supported by many GIS tools.
+If the geometry is lacking SRID a WKB format is produced.
+[Se ST_SetSRID](#ST_SetSRID)
+
+Format: `ST_AsEWKT (A:geometry)`
+
+Since: `v1.1.1`
```
Comment (jiayuasu) [inline docs/api/sql/Function.md:315]: v1.2.1

File: docs/api/sql/Function.md [hunk 4]
```diff
@@ -302,6 +302,24 @@ SELECT ST_AsEWKB(polygondf.countyshape)
 FROM polygondf
 ```
 
+## ST_AsEWKT
+
+Introduction: Return the Extended Well-Known Text representation of a geometry.
+EWKT is an extended version of WKT which includes the SRID of the geometry.
+The format originated in PostGIS but is supported by many GIS tools.
+If the geometry is lacking SRID a WKB format is produced.
+[Se ST_SetSRID](#ST_SetSRID)
```
Comment (jiayuasu) [inline docs/api/sql/Function.md:311]: See ST_SetSRID not 'Se'.  If the geometry is lacking SRID a "WKT" format is produced.

File: docs/api/flink/Function.md [hunk 5]
```diff
@@ -150,4 +150,22 @@ FROM df
...
-Output: `POLYGON ((-0.5 -0.5, 0.5 -0.5, 0.5 0.5, -0.5 0.5, -0.5 -0.5))`
\ No newline at end of file
+Output: `POLYGON ((-0.5 -0.5, 0.5 -0.5, 0.5 0.5, -0.5 0.5, -0.5 -0.5))`
+
+## ST_AsEWKT
+
+Introduction: Return the Extended Well-Known Text representation of a geometry.
+EWKT is an extended version of WKT which includes the SRID of the geometry.
+The format originated in PostGIS but is supported by many GIS tools.
+If the geometry is lacking SRID a WKB format is produced.
```
Comment (jiayuasu) [inline docs/api/flink/Function.md:160]: See ST_SetSRID not 'Se'. If the geometry is lacking SRID a "WKT" format is produced.

**Review by jiayuasu — CHANGES_REQUESTED** [review]

File: flink/src/test/java/org/apache/sedona/flink/FunctionTest.java (hunk 1 above)
Comment (pavan02) [reply flink/src/test/java/org/apache/sedona/flink/FunctionTest.java:85]: Updated. Couldn't find a way to set SRID using flink methods. This test case tests scenario when SRID isn't present. Is that Okay ?

**Review by jiayuasu — APPROVED** [review]

**Pull request merged**